File: backend/app/domain/topics.py
"""Тематики контента платформы."""

TOPIC_PATTERN = r"^(it|auto|russia|sport|postcard)$"

TOPIC_LABELS: dict[str, str] = {
    "it": "IT",
    "auto": "Авто",
    "russia": "Россия",
    "sport": "Спорт",
    "postcard": "Открытки",
}

# Промпты классификации живут в БД (prompt_templates: classification.*),
# дефолты — в app/domain/prompt_defaults.py.

File: frontend/src/constants/topics.js
/** Тематики контента платформы. */

export const TOPIC_LABELS = {
  it: 'IT',
  auto: 'Авто',
  russia: 'Россия',
  sport: 'Спорт',
  postcard: 'Открытки',
}

export const TOPIC_OPTIONS = [
  { value: 'it', label: 'IT' },
  { value: 'auto', label: 'Авто' },
  { value: 'russia', label: 'Россия' },
  { value: 'sport', label: 'Спорт' },
  { value: 'postcard', label: 'Открытки' },
]

/**
 * @param {string | undefined | null} topic
 * @returns {string}
 */
export function topicLabel(topic) {
  return TOPIC_LABELS[topic] || topic || '—'
}
